### ISSUE TITLE
How to see current Renovate version?

### ISSUE THREAD

**Issue by mikaello** [mend/renovate-ce-ee#367]
The Renovate version is often announced in the release notes (in releases), but not always, and sometimes just to the major. To do debugging or determine if a feature is available, it would be nice if the exact version was available. I checked ENVs inside the container, and labels on the image:

```
$ docker inspect whitesource/renovate:5.1.0 --format '{{ json .Config.Labels }}' | jq
{
  "name": "Mend Renovate On-Premises",
  "org.opencontainers.image.ref.name": "ubuntu",
  "org.opencontainers.image.source": "https://github.com/whitesource/renovate-on-prem",
  "org.opencontainers.image.version": "20.04"
}
```

```
$ docker inspect ghcr.io/mend/renovate-ce:6.2.0 --format '{{ json .Config.Labels }}' | jq
{
  "maintainer": "Rhys Arkins <rhys@arkins.net>",
  "org.opencontainers.image.ref.name": "ubuntu",
  "org.opencontainers.image.source": "https://github.com/containerbase/base",
  "org.opencontainers.image.version": "9.20.8"
}
```

... but neither had the Renovate version included.

**Comment by PhilipAbed** [comment]
Are you able to see the logs of each renovate job you run on your CE?

it's printed on every run: https://github.com/renovatebot/renovate/blob/8432d3708c57851e9f0c58a39be4495f4bd87aa5/lib/workers/repository/index.ts#L46

**Comment by mikaello** [comment]
Thanks! Yes, I am able to see the logs, so it works to search for `renovateVersion` in the logs.

It would still be nice to have this as a label on the image though.

**Comment by morremeyer** [comment]
@mikaello This is now also available on the `/api/status` endpoint with 6.5.0, which might help you out. I'll definitely generate a metric out of that.

**Comment by mikaello** [comment]
Having the version in `/api/status` is great, that would definitely help me out. Thanks!
